CONTRIBUTING.md

```diff
@@ -43,12 +43,46 @@
   - PERF: for performance improvements
   - STYLE: a change that does not impact the logic or execution of the code (improve coding style, comments, documentation)
 
-* Ensure the PR adheres to our [coding conventions](#coding-conventions).
-
-* We will review your PR and possibly suggest changes. We also have an automatic build and test system checking all PRs ([Travis-CI](https://travis-ci.org/)). When all lights are green, the PR will be merged.
+The body of the message should clearly describe the motivation of the commit
+(**what**, **why**, and **how**). In order to ease the task of reviewing
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to smooth
+  generation of release notes.
+  2. Try to keep the subject line below 72 characters, ideally 50.
```

Comment:
I would prefer: Try to keep below 73 characters! (So 72 characters is still OK to me.) Problem with 73 or more is that GitHub will then "break" the subject line.

Why would 50 be the ideal? 50 characters is  usually not enough for me, when I do a commit  🤔